packages/coding-standards/src/Sniffs/ForceLateStaticBindingForProtectedConstantsSniff.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\AnnotationHelper;
+use SlevomatCodingStandard\Helpers\ConstantHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+class ForceLateStaticBindingForProtectedConstantsSniff implements Sniff
```

Comment:
Was it too hard to implement as a fixer?

Reply:
It is a fixer. But it uses a PHP_CodeSniffer where everything is a sniff (some are fixable though).